src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,76 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+type TodoListProps = {
+  todos: Todo[];
+  selectedTodo: Todo | null;
+  setSelectedTodo: (selectedTodo: Todo) => void;
+};
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+export const TodoList: React.FC<TodoListProps> = ({
+  todos,
+  selectedTodo,
+  setSelectedTodo,
+}) => {
+  const completedTodo = (
```

Comment:
We can't do it like that in React. 
If you want to go that way, there should be a separate component or a helper function, like `renderCompletedTodo =() => `. 

That's a good idea, but in this case my point was just to split the ternary expression you had into 2 chunks:
`todo.completed ? <></> : <></> `=> 
```
todo.completed && <></>
!todo.completed && <></>
```

If you want to take it step further - sure, that sounds like a plan, but we have to do it properly.